src/components/SecondaryPanes/Breakpoints/Breakpoints.css

```diff
@@ -20,11 +16,18 @@
   display: flex;
   width: 100%;
   align-items: center;
+  padding-top: 3px;
+  padding-bottom: 3px;
+}
+
+.breakpoints-list .breakpoint-heading:nth-child(2) {
```

Comment:
This `:nth-child(2)` looks a bit too circumstantial, and prone to breakage if we add more elements. Not sure what it does, but perhaps a `:not-first-child` or `:nth-child(n+2)` would work too?

Reply:
This was a way around having to change any of the Component code to add 4px to the start and end of the actual list of breakpoints. This code snippet:

```
.breakpoints-list .breakpoint-heading:nth-child(2) {
  margin-top: 4px;
}
.breakpoints-list .breakpoint:last-child {
  margin-bottom: 4px;
}
```

- adds a 4px margin-top to the start of the breakpoints list, which starts with a breakpoint heading div
- adds a 4px margin-bottom to the end of the breakpoints list, which ends with a breakpoint div

The problem I ran into was that the `.breakpoints-list` div also includes `.breakpoints-exceptions` as its first child, which prevented me from just adding a padding-top+bottom to `.breakpoints-list.`

An alternative is to move `.breakpoints-exceptions` out of `breakpoints-list` so I can just set a top+bottom padding for `breakpoints-list` to do the same thing. 

```html
// BEFORE
<div class="breakpoints-list">
  <div class="breakpoints-exceptions"> ... </div>
  <div class="breakpoint-heading"> ... </div>
  <div class="breakpoint"> ... </div>
</div>

// AFTER
<div>
  <div class="breakpoints-exceptions"> ... </div>
  <div class="breakpoints-list">
    <div class="breakpoint-heading"> ... </div>
    <div class="breakpoint"> ... </div>
  </div>
</div>
```